pkg/features/antrea_features.go

```diff
@@ -158,6 +158,10 @@ const (
 	// alpha: v1.15
 	// Enable layer 7 flow export on Pods and Namespaces
 	L7FlowExporter featuregate.Feature = "L7FlowExporter"
+
+	// alpha: v1.15
+	// Enable the PingMonitoringTool feature.
+	PingMonitoringTool featuregate.Feature = "PingMonitoringTool"
```

Comment:
let's think about some better names for this. Maybe `LatencyMonitor` or `ConnectivityMonitor` (`LatencyMonitor` may be a bit too specific especially if we start adding extra functionality).

@Dyanngg @heanlan

Reply:
`LatencyMonitor` sounds good to me. Do you think we should add `Node` prefix though?

Reply:
I am fine with the `Node` prefix.

Reply:
I have updated it yet.